contributors/devel/contributor-cheatsheet.md

```diff
@@ -0,0 +1,39 @@
+# Kubernetes
+
+| Repo | PRs | Issues | Notes |
+| ---- | --- | ------ | ----- |
+| [k/kubernetes](https://github.com/kubernetes/kubernetes) | [PRs](https://github.com/kubernetes/kubernetes/pulls) | [Issues](https://github.com/kubernetes/kubernetes/issues) | [Meeting Notes](http://bit.ly/kubenotes)
```

Comment:
Here and below two places - with "Kubernetes" header (LOC # 1) if we don't use "k/" that should be OK as it's self readable. Just a thought, it's up to you :-) Thanks!

Reply:
+1 for no K